tests/D2L.CodeStyle.Analyzers.Test/Specs/RequireNamedArgumentsAnalyzer.cs

```diff
@@ -276,6 +276,16 @@ public SomeInheritedClass( int b1, int b2, int b3, int b4, int b5, string _ )
 
 			public SomeInheritedClass( int b1, int b2, int b3, int b4, int b5, string _ )
 				/* TooManyUnnamedArgs(5) */ : base( b1, b2, b3, b4, b5 ) /**/{ }
-		}
-    }
+		}
+	}
+
+	public static class PR659 {
+		public interface IFoo { }
+		public class Foo : IFoo {
+			public Foo( long a, long b ) { }
+			static void SomeMethod( long a, long b, out IFoo c ) {
+				c = new Foo( a, b );

```

Comment:
Took a minute to figure out, but it's about being converted to interfaces, since interfaces have no basetype. I don't fully know that the `out` parameter is important, but it kinda seems like it was in order for the parent syntax to have a type